test/models/resource_test.rb

```diff
@@ -33,6 +33,9 @@ class ResourceTest < ActiveSupport::TestCase
       assert_equal ["Communication Skills for Dummies", "How to Fail in Business", "How to Succeed in Business", "Public Speaking 101 Workshop"], Resource.alphabetical.map  { |e| e.title }
     end
 
+    should "show link without 'http://' is reformated" do
+      assert_equal "http://www.google.com", @comm_dumm.link
```

Comment:
Can you write a test that tests http:// won't be added to a link, if it already includes it